charts/admission-controller/templates/scanner/deployment.yaml

```diff
@@ -73,31 +74,31 @@ spec:
           env:
             {{- if (or .Values.scanner.httpProxy .Values.scanner.httpsProxy) }}
```

Comment:
@ironashram I believe you should change also this line otherwise the proxy will get set only when using `admissionController.scanner.httpProxy` and gets ignored when using the global settings
cc: @lilx1ao @aroberts87

Reply:
indeed, great catch thank you

Reply:
Thanks for fixing it Michele! 

One more question, I see you have also set the namespace variable for multiple objects, like: secrets, service, role, PodSecurityPolicy, etc.. 

What's the purpose of that? You would like to give the possibility to deploy the admission controller on a different namespace while using `sysdig-deploy`? I'm asking because by default without specifying it, helm will take care of patching the manifest files and add the namespace specified with `helm install` command

Reply:
the idea was to give that option yes, but also to try and align with the other charts which are already among the dependencies